x/observer/client/cli/query_tss_fund_migrator.go

```diff
@@ -0,0 +1,63 @@
+package cli
+
+import (
+	"context"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/spf13/cobra"
+
+	"github.com/zeta-chain/zetacore/x/observer/types"
+)
+
+func CmdGetTssFundsMigrator() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "show-tss-funds-migrator [chain-id]",
```

Comment:
Using `show` for single and `list` for multiple, as this seems to be the general terminology across the code. 
We can consider moving to `get` and `getAll`, which is a bit easier to understand

Reply:
Show sounds better to me